### ISSUE TITLE
Can't change font color and other css properties

### ISSUE THREAD

**Issue by maraoz** [nCrazed/invoice-ninja#5]
I'm trying to use invoice-ninja to generate nice invoices for my app, but I'm finding it difficult to customize the design. 

I tried changing the title color but no matter what I do it stays black. Any pointers to how to do this? thanks in advance!

**Comment by nCrazed** [comment]
Been a while since I've touched this so not quite sure my self. Best starting point is probably markdown-pdf docs

**Comment by maraoz** [comment]
Yeah, I already looked, but I thought you might have some advice here. Thanks

**Comment by nCrazed** [comment]
I'll take a closer look when I get some free time (probably next week).
